options.go

```diff
@@ -149,6 +149,9 @@ type grOptions struct {
 	GoroutineTriggerPercentDiff int // goroutine trigger diff in percent
 	GoroutineTriggerNumAbs      int // goroutine trigger abs in number
 
+	// if current goroutine number is greater than GoroutineTriggerNumMax
+	// holmes would not dump goroutine profile.
+	GoroutineTriggerNumMax int
```

Comment:
Move to the next row of `GoroutineTriggerNumMin`, and modfiy the  comment. keep the same style of code.

Reply:
//   1. goroutine_num > GoroutineTriggerNumMin && goroutine_num < GoroutineTriggerNumMax && goroutine diff percent > GoroutineTriggerPercentDiff